torch/special/__init__.py

```diff
@@ -505,6 +505,32 @@
 Alias for :func:`torch.round`.
 """)
 
+log_softmax = _add_docstr(_special.special_log_softmax,
+                          r"""
+Computes softmax followed by a logarithm.
+
+While mathematically equivalent to log(softmax(x)), doing these two
+operations separately is slower, and numerically unstable. This function
+uses an alternative formulation to compute the output and gradient correctly.
```

Comment:
I think this sentence can be removed? It's not clear what this is an alternative to